### DISCUSSION TITLE
Tips on earning trust in SolidJS from FrontEnd community

### DISCUSSION THREAD

**Discussion by trivikr** [solidjs/solid#618] [General]
This generic question is created to collect suggestions on how to gain more trust in SolidJS from FrontEnd community.
This is a follow-up to the video [Is SolidJS better than React?](https://www.youtube.com/watch?v=Lygh1pYLz9A) on YouTube.

Tips to contribute:
* Go through existing comments. Upvote and/or add a reply if your suggestion is already shared so that we know what's more important for the community.
* If you have a new idea, do add a new comment for your suggestion. If your idea needs to be tracked separately, create an issue or discussion and reference it.
* Limit the conversations/replies specific to the idea/suggestion being discussed.

**Comment by trivikr** [comment]
Move all solidjs modules under `@solidjs` npm scope.

**Reply by trivikr** [reply]
This suggestion is being tracked at https://github.com/solidjs/solid/discussions/611

**Comment by trivikr** [comment]
Add Code Of Conduct for SolidJS projects.

**Reply by trivikr** [reply]
This suggestion is being tracked at https://github.com/solidjs/solid/pull/607

**Comment by trivikr** [comment]
Add Contributing Guidelines in `CONTRIBUTING.md` for SolidJS projects.

**Reply by trivikr** [reply]
This suggestion was discussed in [Discord Thread](https://discord.com/channels/722131463138705510/877571172588482570).

**Comment by illiaChaban** [comment]
Add JSDoc to exported methods + possibly url pointing to the documentation online.

**Reply by illiaChaban** [reply]
The types are pretty good and there was nothing I couldn't figure out just by looking at types so far. However, I did find some of the documentation very succinct that sometimes didn't fully explain how the method / property should work.
I also believe JSDoc can help with the speed of development, so you don't have to constantly lookup documentation

Love the library so far!

**Reply by atk** [reply]
We're already working on improving the developer experience. The next release will have JSDoc tooltips for all signals and related primitives; stores are next.
